CHANGELOG.md

```diff
@@ -1,16 +1,61 @@
-## 3.0
+## 4.1.0
+
+* Allow getGrades() without a line item ([#34](https://github.com/packbackbooks/lti-1-3-php-library/pull/34))
+* Fix fetching of line items and eliminate a PHP warning for missing key ([#35](https://github.com/packbackbooks/lti-1-3-php-library/pull/35))
+* Include resourceLinkId attribute when creating a line item ([#36](https://github.com/packbackbooks/lti-1-3-php-library/pull/36))
+
+## 4.0.0
+
+* Added a new method to `ILtiServiceConnector`: `setDebuggingMode()`. ([#32](https://github.com/packbackbooks/lti-1-3-php-library/pull/32))
+
+## 3.0.3
+
+* Adds response/request logging to LtiServiceConnector ([#31](https://github.com/packbackbooks/lti-1-3-php-library/pull/31))
```

Comment:
super minor nitpick but especially with public facing documentation/changelogs we should keep it consistently either past or present tense

Reply:
Great feedback, updated!